cmd/internal/ecs/ecs.go

```diff
@@ -115,6 +127,16 @@ func init() {
 	Cmd.PersistentFlags().StringVar(&awsCredentialsFlag, "aws-credentials", "", "Location of AWS credentials file.")
 	Cmd.PersistentFlags().MarkHidden("aws-credentials")
 
+	// initialize apidump flags as persistent flags for the ecs command
+	Cmd.PersistentFlags().StringVar(&filterFlag, "filter", "", "Used to match packets going to and coming from your API service.")
+	Cmd.PersistentFlags().StringSliceVar(&hostAllowlistFlag, "host-allow", nil, "Allows only HTTP hosts matching regular expressions.")
+	Cmd.PersistentFlags().StringSliceVar(&hostExclusionsFlag, "host-exclusions", nil, "Removes HTTP hosts matching regular expressions.")
+	Cmd.PersistentFlags().StringSliceVar(&interfacesFlag, "interfaces", nil, "List of network interfaces to listen on. Defaults to all interfaces on host.")
+	Cmd.PersistentFlags().StringSliceVar(&pathAllowlistFlag, "path-allow", nil, "Allows only HTTP paths matching regular expressions.")
+	Cmd.PersistentFlags().StringSliceVar(&pathExclusionsFlag, "path-exclusions", nil, "Removes HTTP paths matching regular expressions.")
+	Cmd.PersistentFlags().Float64Var(&rateLimitFlag, "rate-limit", apispec.DefaultRateLimit, "Number of requests per minute to capture.")
+	Cmd.PersistentFlags().IntVar(&statsLogDelay, "stats-log-delay", apispec.DefaultStatsLogDelay_seconds, "Print packet capture statistics after N seconds.")
```

Comment:
Omit if undocumented or hidden.

Possibly, make hidden -- this is not something we would normally expect somebody to do when running noninteractively.

Reply:
Instead of marking them as hidden, we should document these flags in LC docs.
Users can use these flags while running `ecs add` for their ECS tasks